src/server/modules/sendResponse.ts

```diff
@@ -16,6 +16,11 @@ import {
 } from "node-hl7-client/hl7";
 import type { ListenerOptions } from "../../utils/normalize.js";
 import { MLLPCodec } from "../../utils/codec.js";
+import { HL7ServerError } from "../../utils/exception.js";
+
+const MSA_1_VALUES_v2_1 = ["AA", "AR", "AE"];
+const MSA_1_VALUES_v2_x = ["CA", "CR", "CE"];
+type ValidMSA1 = "AA" | "AR" | "AE" | "CA" | "CR" | "CE";
```

Comment:
I think this is still to be a lower case v

Reply:
Ah ok, I can change it, but TS convention is to use PascalCase for types/interfaces

Reply:
Ok. I will check it out. I am using camelcase. :)